server/controllers/temporary-accommodation/manage/reportsController.ts

```diff
@@ -34,6 +34,12 @@ export default class ReportsController {
     }
   }
 
+  new(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      return res.redirect(301, paths.reports.index({}))
+    }
+  }
```

Comment:
It may be nice to think of a way of flagging this as deprecated -- maybe renaming the method would be sufficient, or creating a generic decorator that would handle the redirect? It seems we'll be needing this in a couple of places, and it would help any new dev understand the reason behind the redirect.

Reply:
Yes, that seems like the best option -- less code when new redirects are needed, and the concern of routing is all kept in one place. Better!